lib/workspace.rb

```diff
@@ -0,0 +1,69 @@
+require_relative "channel"
+require_relative "user"
+require_relative "recipient"
+require "dotenv"
+require "httparty"
+require "pry"
+Dotenv.load
+
+module Slack
+  class Workspace
```

Comment:
This works as a collection of class methods, but if you're using all class methods, you might as well just make `Workspace` a module.  Instead I would suggest making `Workspace` a class with the attributes `@users`, `@channels` and `@selected`, which means you could keep track of the selected user or channel in your main app.